bin/dep

```diff
@@ -29,6 +29,17 @@ if (isset($options['f'])) {
     $userSpecifiedFile = $options['file'];
 }
 
+if(empty($userSpecifiedFile)) {
+    $fOptIndex = array_search('-f', $argv);
+    $fileOptIndex = array_search('--file', $argv);
```

Comment:
You could use the ternary here instead of the if statement:
```php
$userSpecifiedFile = $fOptIndex ? $argv[$fOptIndex+1] : $argv[$fileOptIndex+1];
```